CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.1-RC1
+
+## Enhancements
+
+- Support for Alpine Linux.
+- New LogLevelType enum and functions to convert between different log level type levels.
+- Added netstandard20 as a target.
+
```

Comment:
I think you should either mention/link librdkafka v1.0.1, or copy-paste the fixes from v1.0.1, the latter being easy for maintenance releases, but not really viable for feature releases.